src/python/WMCore/WMExceptions.py

```diff
@@ -6,15 +6,15 @@
 mappings to a human readable message.
 """
 
-WMEXCEPTION = {'WMCore-1' : 'Not allowed to instantiate ',
-               'WMCore-2' : 'Problem creating database table ',
-               'WMCORE-3' : 'Could not find in library class ',
-               'WMCORE-4' : 'Problem with loading the class ',
-               'WMCORE-5' : 'Dialect not specified in configuration ',
-               'WMCORE-6' : 'Message name is same as diagnostic message name! ',
-               'WMCORE-7' : 'Component name is reserved word. ',
-               'WMCORE-8' : 'No config section for this component! ',
-               'WMCORE-9' : 'Problem inserting a trigger flag. ',
+WMEXCEPTION = {'WMCore-1': 'Not allowed to instantiate ',
+               'WMCore-2': 'Problem creating database table ',
```

Comment:
We should try to understand why this case shift is here and fix it. WMCore-2 is widely used. WMCore-1 is not used at all.